docs/csharp/toc.yml

```diff
@@ -892,16 +904,18 @@
     - name: Serialization (C#)
       items:
       - name: Overview
+        displayName: serialization
         href: programming-guide/concepts/serialization/index.md
       - name: "How to write object data to an XML file"
         href: programming-guide/concepts/serialization/how-to-write-object-data-to-an-xml-file.md
       - name: "How to read object data from an XML file"
         href: programming-guide/concepts/serialization/how-to-read-object-data-from-an-xml-file.md
       - name: "Walkthrough: Persisting an Object in Visual Studio"
         href: programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md
-  - name: Statements, Expressions, and Operators
+  - name: Statements, expressions, and operators
     items:
     - name: Overview
+      displayName: statements expressions operators
```

Comment:
```suggestion
      displayName: Statements, expressions, and operators
```

Reply:
Here, I think the commas may be necessary for the drop down to find each one separately. But I'm not sure. We needed them when we added multi-character operators.